app/presenters/instance_presenter.rb

```diff
@@ -26,6 +26,12 @@ def domain_count
     Rails.cache.fetch('distinct_domain_count') { Account.distinct.count(:domain) }
   end
 
+  def open_registrations
```

Comment:
That was what I had originally, but changed in response to this review: https://github.com/tootsuite/mastodon/pull/1575#pullrequestreview-32281280

If I move the test to RegistrationsController then it has to be duplicated in AboutController; it's in InstancePresenter to avoid that duplication.